BasicController.java

```diff
@@ -25,7 +25,7 @@ public ApiResult add(@RequestParam BigDecimal augend, @RequestParam BigDecimal a
 
    @GetMapping("/subtract")
    public ApiResult subtract(@RequestParam BigDecimal minuent, @RequestParam BigDecimal subtrahend) {
```

Comment:
Incorrect return type in the subtract method. The method should return an `ApiResult` object, but the modified code attempts to return a `BigDecimal` directly.